dev/tests/integration/testsuite/Magento/Checkout/_files/quote_with_multiple_items.php

```diff
@@ -0,0 +1,65 @@
+<?php
```

Comment:
If we create new fixtures for GraphQL api-functional tests, I would strongly recommend using a dedicated place (directory) for that. Please, check the following fixtures as the example 

https://github.com/magento/magento2/tree/a5a98dfa56cac8cdb60e15568c10578f522be3f3/dev/tests/integration/testsuite/Magento/GraphQl/Quote/_files

(please, take a look at the files path)